src/org/omegat/filters2/html2/FilterVisitor.java

```diff
@@ -163,12 +163,40 @@ public void visitTag(Tag tag) {
 
         boolean keepIntact = isProtectedTag(tag);
 
+        // if (tag.getText().equalsIgnoreCase("style")) {
+        Log.log("************     ************");
+        Log.log("************ " + tag.getText() + " ************");
+        Log.log("************     ************");
+        Log.log("recurseSelf " + recurseSelf);
+
+        Log.log("recurseChildren " + recurseChildren);
+
+        /** Do we collect the translatable text now. */
+        Log.log("isTextUpForCollection " + isTextUpForCollection);// = false;
+
+        /**
+         * Did the PRE block start (it means we mustn't compress the spaces).
+         */
+        Log.log("preformatting " + preformatting);// = false;
+        Log.log("hasAnEndTag " + hasAnEndTag(tag));// = false;
+
+        // }
```

Comment:
looks like debug messages are written out as INFO category.
It should be `Log.debug()` or somethings similar.